### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Loui4**

### CODE REVIEW

File: public/countryphones.json
Comment (dominickasanga) [inline public/countryphones.json:1]: file removed?..

File: src/router/index.ts [hunk 1]
```diff
@@ -44,11 +43,7 @@ const routes: Array<RouteRecordRaw> = [
         name: "OfflineRecords",
         component: OfflineRecords,
     },
-    {
```
Comment (dominickasanga) [inline src/router/index.ts:47]: reomoved line?

File: src/services/appointment_service.ts [hunk 2]
```diff
@@ -9,8 +9,8 @@ export class AppointmentService extends AppEncounterService {
         return AppEncounterService.getJson(`/programs/${this.programID}/patients/${this.patientID}/next_appointment_date`, { date: this.date });
     }
 
-    static async getDailiyAppointments(date: any, end_date = '', srch_text = '') {
+    static async getDailiyAppointments(date: any) {
         const programID = AppEncounterService.getProgramID();
-        return AppEncounterService.getJson(`/programs/${programID}/booked_appointments`, { date, end_date, srch_text, paginate: false });
+        return AppEncounterService.getJson(`/programs/${programID}/booked_appointments`, { date: date, paginate: false });
```
Comment (dominickasanga) [inline src/services/appointment_service.ts:14]: end point requires the removed params..

File: src/utils/GeneralUti.ts [hunk 3]
```diff
@@ -68,15 +69,12 @@ export function combineArrays<T>(...arrays: T[][]): T[] {
...
-
-let baseURL = getBaseURl();
-if (baseURL.length > 0) {
-    baseURL = "/" + baseURL;
+    return date1.getFullYear() === date2.getFullYear() &&
+           date1.getMonth() === date2.getMonth() &&
+           date1.getDate() === date2.getDate();
 }
 
-const configPaths = [`${baseURL}/config.json`];
```
Comment (dominickasanga) [inline src/utils/GeneralUti.ts:79]: uses dymaic base URL

File: vite.config.ts [hunk 4]
```diff
@@ -5,8 +5,7 @@ import { defineConfig } from "vite";
 import { visualizer } from "rollup-plugin-visualizer";
 import basicSsl from "@vitejs/plugin-basic-ssl";
 
-const baseURL = "mahis";
-
+const baseURL = '/mahis/'
```
Comment (dominickasanga) [inline vite.config.ts:8]: app breaks if.. use the .. exclude the '/'

**Review by dominickasanga — DISMISSED** [review]
looks good

**Review by petroskayange — APPROVED** [review]

**Pull request merged**